pkg/manager/member/tidbcluster_status_manager.go

```diff
@@ -108,7 +108,7 @@ func (tcsm *TidbClusterStatusManager) syncDashboardMetricStorage(tc *v1alpha1.Ti
 	if tc.Spec.PD == nil {
 		return nil
 	}
-	pdEtcdClient, err := tcsm.pdControl.GetPDEtcdClient(pdapi.Namespace(tc.Namespace), tc.Name, tc.IsTLSClusterEnabled())
+	pdEtcdClient, err := tcsm.pdControl.GetPDEtcdClient(pdapi.Namespace(tc.Namespace), tc.Name, tc.Spec.ClusterDomain, tc.IsTLSClusterEnabled())
```

Comment:
No need to sync L63 and L67 if `spec.cluster` is configured.
That's for TiDB Dashboard, we should address this when the monitoring solution is ready for the across Kubernetes deployment.

Reply:
This comment is not addressed.